### PR TITLE
zipp: add recipe for v0.3.3

### PR DESCRIPTION
**Pull request by nehaljwani**

### CODE REVIEW

**Comment by nehaljwani** [comment]
@conda-forge/help-python What's with the windows failure here, on azure pipelies?

**Comment by beckermr** [comment]
You can ignore failures on azure for now.

File: recipes/zipp/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+test:
+  imports:
+    - zipp
+
+about:
+  home: https://github.com/jaraco/zipp
+  license: MIT
+  license_family: MIT
+  license_file: LICENSE
+  summary: Summary of the package
```
Comment (beckermr) [inline recipes/zipp/meta.yaml]: Please fill this out.

Comment (nehaljwani) [reply recipes/zipp/meta.yaml]: Done.

**Pull request merged**